libs/core/src/lib/card/card-footer.component.html

```diff
@@ -0,0 +1,10 @@
+<ng-content></ng-content>
+<div class="fd-card__footer--start">
```

Comment:
Where are these classes coming from? They don't follow BEM. In order to have the modifier `fd-card__footer--start` you need a base class `fd-card__footer` on the same element. For example:
```
<div class="fd-card__footer fd-card__footer--start">...</div>
```

Reply:
Those classes are not even necessary, because __footer does `space-between`. I messed up modifier with element